os-datavirt/added/standalone-openshift.xml

```diff
@@ -525,14 +525,14 @@
             <preparedplan-cache infinispan-container="teiid-cache"/>
             <transport name="embedded"/>
             <transport name="odata">
-                <authentication security-domain="teiid-security"/>
+                <authentication security-domain="##ODATA_SECURITY_DOMAIN##"/>
```

Comment:
I was speaking to it being generic and jdv is a first class consumer, defining transports for odata, etc.